### PR TITLE
Jinghan/refresh informer for all non performance-critical oomstore APIs

### LINKED ISSUE #591: Refresh informer in non performance-critical read paths

### PR DESCRIPTION
**Pull request by jinghancc**
This PR does:
- refresh informer at the beginning of all non performance-critical APIs

close #591

### CODE REVIEW

**Review by wfxr — APPROVED** [review]
lgtm

File: pkg/oomstore/apply.go [hunk 1]
```diff
@@ -165,7 +168,7 @@ func (s *OomStore) applyFeature(ctx context.Context, txStore metadata.WriteStore
 	return nil
 }
 
-func buildApplyStage(ctx context.Context, opt apply.ApplyOpt) (*apply.ApplyStage, error) {
+func BuildApplyStage(ctx context.Context, opt apply.ApplyOpt) (*apply.ApplyStage, error) {
```
Comment (YikSanChan) [inline pkg/oomstore/apply.go:171]: @lianxmfor fyi

**Review by YikSanChan — APPROVED** [review]
lgtm

File: pkg/oomstore/apply.go (hunk 1 above)
Comment (lianxmfor) [reply pkg/oomstore/apply.go:171]: The reason `buildApplyStage` is private is that I don't want to expose the oomstore layer to the public, but being private would also result in the test files having to be under this package as well.
If you feel comfortable with the oomstore layer exposing x to the public, that's fine

**Review by lianxmfor — APPROVED** [review]
lgtm

File: pkg/oomstore/apply.go (hunk 1 above)
Comment (jinghancc) [reply pkg/oomstore/apply.go]: discussed offline, we decided to keep the function private

**Pull request merged**